sonar-batch/src/main/java/org/sonar/batch/bootstrap/GlobalContainer.java

```diff
@@ -136,6 +158,35 @@ public void executeAnalysis(Map<String, String> analysisProperties, Object... co
     new ProjectScanContainer(this, props, components).execute();
   }
 
+  private static void triggerViews(ServerClient serverClient, Server server) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused private "triggerViews" method. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUnusedPrivateMethod)
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) The Cyclomatic Complexity of this method "triggerViews" is 11 which is greater than 10 authorized. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AMethodCyclomaticComplexity)